tez-dag/src/test/java/org/apache/tez/dag/app/rm/TestTaskSchedulerManager.java

```diff
@@ -839,6 +855,60 @@ protected void instantiateSchedulers(String host, int port, String trackingUrl,
     }
   }
 
+  @Test(timeout = 10000)
+  public void testHandleException() throws Exception {
+    Configuration tezConf = new Configuration(new YarnConfiguration());
+    UserPayload defaultPayload = TezUtils.createUserPayloadFromConf(tezConf);
+
+    // Parse plugins
+    List<NamedEntityDescriptor> tsDescriptors;
+    BiMap<String, Integer> tsMap;
+    tsDescriptors = Lists.newLinkedList();
```

Comment:
you can merge initial values with declarations here

Reply:
Okay, I will fix it.

Reply:
thanks, also if you can fix any of these:
https://ci-hadoop.apache.org/job/tez-multibranch/job/PR-236/1/artifact/out/diff-checkstyle-tez-dag.txt
```
./tez-dag/src/test/java/org/apache/tez/dag/app/rm/TestTaskSchedulerHelpers.java:164:    public TaskSchedulerManagerForTest(AppContext appContext,:5: Redundant 'public' modifier. [RedundantModifier]
./tez-dag/src/test/java/org/apache/tez/dag/app/rm/TestTaskSchedulerManager.java:908:    waitFor(() -> { return eventHandler.getEventSize() > 0;}, 1000, 5000);:19: '{' at column 19 should have line break after. [LeftCurly]
./tez-dag/src/test/java/org/apache/tez/dag/app/rm/TestTaskSchedulerManager.java:908:    waitFor(() -> { return eventHandler.getEventSize() > 0;}, 1000, 5000);:59: ';' is not followed by whitespace. [WhitespaceAfter]
```

Reply:
I will fix it. But ParameterNumber check is not easy to fix.

```
./tez-dag/src/main/java/org/apache/tez/dag/app/DAGAppMaster.java:2626:  public static void parseAllPlugins(:22: More than 7 parameters (found 9). [ParameterNumber]
```